### PR TITLE
Attributions

### PR DESCRIPTION
**Pull request by JustinRMoy**
Added the js file for the attributions page

### CODE REVIEW

File: src/pages/Attributions.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export default function Schedule() {
```
Comment (mbruckert) [inline src/pages/Attributions.js]: Hey Justin! Please change the function name to match the name of the page. Other than that, good work!

**Review by mbruckert — CHANGES_REQUESTED** [review]

File: src/pages/Attributions.js (hunk 1 above)
Comment (JustinRMoy) [reply src/pages/Attributions.js]: I just updated it

**Review by nategaulke — APPROVED** [review]
Looks good to me!

**Review by mbruckert — CHANGES_REQUESTED** [review]
Make sure that you import the Attributions function in App.js

**Review by mbruckert — APPROVED** [review]
Looks good! You can merge

**Pull request merged**